torch/csrc/jit/runtime/serialized_shape_function_registry.cpp

```diff
@@ -898,8 +898,8 @@ def mv(self: List[int],
   _1 = "AssertionError: max_pool2d: stride must either be omitted, a single int, or a tuple of two ints"
```

Comment:
can we leave out this file? it's codegen-ed so these changes will be removed the next time someone regenerates the file.

Reply:
Sure, just reverted now.